src/Task/domain/entity/TaskInterface.ts

```diff
@@ -2,8 +2,28 @@ export interface TaskInterface {
   taskId?: string;
   title: string;
   description: string;
-  user: {
-    userId: string;
-    fullName: string;
-  };
+  user: User;
+  category: string;
+  location?: string;
+  budget: number;
+  scheduling: SchedulingOption;
+  specificDate?: Date;
+  timeslot?: TimeSlot;
+  createdon?: Date;
+}
+
+export enum SchedulingOption {
+  Flexible = 'flexible',
```

Comment:
enum usually use ALL_CAPS_WITH_UNDERSCORE

Reply:
fixed it, please take a look.